doc/source/advanced.rst

```diff
@@ -51,6 +51,20 @@ And vary the number of return values for tasks (and actor methods too):
     assert ray.get(id1) == 0
     assert ray.get(id2) == 1
 
+And specify a name for the task (and actor methods too) at task submission time:
+
+.. code-block:: python
+
+   @ray.remote
+   def f(x):
+      return x + 1
+
+   obj = f.options(name="special_f").remote(3)
+   assert ray.get(obj) == 4
```

Comment:
@rkooo567 I'm happy to change this assertion to something more sensical, but what do you mean by "insert an image"?

Reply:
Ah, so I thought showing ray.get doesn't really have any meaning in this context (since people already knew how this works). Instead, to demonstrate what the name is doing, we can insert an image of dashboard of ps to show that the assigned name is properly set on each process.

Reply:
Ah, ok! I was planning on adding `assert setproctitle.getproctitle() == "ray::special_f"` within the task to demonstrate that the process name will be properly set. Would you prefer this, a dashboard image, or both?

Reply:
I think both will be great in this case. If I have to choose one, I prefer the image because the behavior is more obvious in my opinion! (But you might have better opinion here since you are an actual user!)